packages/core-transaction-pool-redis/__tests__/connection.test.js

```diff
@@ -21,6 +23,10 @@ afterAll(async () => {
   await app.tearDown()
 })
 
+beforeEach(async () => {
```

Comment:
What is the reason for this change? We already have

```
afterEach(async () => {
  await connection.flush()
})
```
doing the same at `beforeEach` looks unnecessary / redundant. Maybe I miss something?